### PR TITLE
.coafile: Add YAMLLintBear

### LINKED ISSUE #31: Check repository for .yml files using YAMLLintBear
Add YAMLLintBear in .coafile and fix .yml file issues raised by coala. See https://github.com/coala/coala-bears/pull/1799 for reference.
difficulty/newcomer

Opened via [gitter](https://gitter.im/undefined/?at=5945cee46462d8493c224f13) by @Monal5031

### PR DESCRIPTION
**Pull request by khanchi97**
Add YAMLLintBear for fixing issues with .yml files

Closes https://github.com/coala/coala-vim/issues/31

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -20,3 +20,7 @@ default_actions = SpaceConsistencyBear: ApplyPatchAction
 bears = GitCommitBear
 shortlog_trailing_period = False
 shortlog_regex = ([^:]*|\S+: [A-Z0-9*].*)
+
+[yml]
+bears = YAMLLintBears
```
Comment (hemangsk) [inline .coafile]: YAMLLintBear*

**Review by hemangsk — CHANGES_REQUESTED** [review]

File: .coafile (hunk 1 above)
Comment (khanchi97) [reply .coafile]: what i should do ?
1. modify it and make another PR
2. you want to suggest.

Comment (hemangsk) [reply .coafile]: You can do modifications to commit, and in this PR only. Please have a look on the process here http://api.coala.io/en/latest/Developers/Git_Basics.html#follow-up  :) 👍

Comment (hemangsk) [reply .coafile]: Marking ```work in progress```

**Comment by vidurkatyal** [comment]
The issue reference line should say 'Closes' instead of 'Fixes' as this commit is adding an enhancement.
Refer the [Writing Good Commits](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference) guide. :smile:

**Comment by khanchi97** [comment]
thank you vidurkatyal

**Review by vidurkatyal — APPROVED** [review]

**Comment by vidurkatyal** [comment]
ack 3b86a63a7c175f85185d9c08ecfb6e9d6f97b80c

File: .coafile (hunk 1 above)
Comment (khanchi97) [reply .coafile]: I have changed it.

**Comment by prachi1210** [comment]
LGTM 
ack 3b86a63

**Comment by SanketDG** [comment]
@rultor merge

**Pull request merged**